src/sniffer/CmdHandler.c

```diff
@@ -0,0 +1,35 @@
+
+#include "CmdHandler.h"
+
+#include <stdio.h>
+
+#include "ErrorHandler.h"
+
+CmdHandler_status_t CmdHandler_ProcessCommand(CmdHandler_cmdCode_t command) {
+    switch(command) {
+        case CmdHandler_cmdCode_EXIT: {
+            ErrorHandler_CleanExit();
+            break;
+        }
+
+        case CmdHandler_cmdCode_IDLE: {
+            printf("IDLE!\n");
+            break;
+        }
+
+        default: {
+            return CmdHandler_status_UNKNOWN_COMMAND;
+        }
+    }
+
+    return CmdHandler_status_OK;
+}
+
+void CmdHandler_PrintCommandList() {
```

Comment:
```suggestion
void CmdHandler_PrintCommandList(void) {
```
Just to match the .h declaration.

Reply:
I think you can only set the parameter as void in function definitions, the compiler does not accept a paramiter void in the implementation.